recipes/mp-units/2.2.0/conanfile.py

```diff
@@ -0,0 +1,266 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import default_cppstd, valid_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+
+required_conan_version = ">=1.59.0"
+
+
+def loose_lt_semver(v1, v2):
```

Comment:
If we end up only supporting Conan 2, this can be removed in favour of `conan.tools.scm.Version`, which correctly handles things like this :)
(Conan 1 implementation is a bit more fragile/strict, so we would probably still need it!)

Reply:
Nice! (Finally :wink:)